userscripts/Custom Race Presets (Torn PDA).js

```diff
@@ -33,15 +33,15 @@ var presets = [{
 	numberOfLaps: 1,
 	upgradesAllowed: true,
 	betAmount: 0,
-	waitTime: 1,
+	waitTime: "ASAP",
 	password: "",
 },
 {
 	name: "1hr Start - Docks",
 	maxDrivers: 100,
 	trackName: "Docks",
 	numberOfLaps: 100,
-	waitTime: 60,
+	waitTime: "Today 23:45 TCT",
```

Comment:
We can't just hardcode a random time here. Leave the waitTime as the number of minutes, and we'll handle it down below.